install_config/install/docker_registry.adoc

```diff
@@ -1275,6 +1328,33 @@ To prevent this, avoid the tagging of internally managed images completely, or
 re-push the built image to the desired namespace
 link:#access-pushing-and-pulling-images[manually].
 
+[[known-issue-s3-image-push-fails]]
+==== Image Push fails with `500 Internal Server Error` on S3 storage
+
+There are problems reported happening when the registry runs on S3 storage
+back-end. Push to a docker registry occasionally fails with the following
+error:
+
+----
+Received unexpected HTTP status: 500 Internal Server Error
+----
+
+To debug this, you need to link:#viewing-logs[view the registry logs]. In there,
+look for similar error messages occurring at the time of the failed push:
+
+----
+time="2016-03-30T15:01:21.22287816-04:00" level=error msg="unknown error completing upload: driver.Error{DriverName:\"s3\", Enclosed:(*url.Error)(0xc20901cea0)}" http.request.method=PUT
+...
+time="2016-03-30T15:01:21.493067808-04:00" level=error msg="response completed with error" err.code=UNKNOWN err.detail="s3: Put https://s3.amazonaws.com/oso-tsi-docker/registry/docker/registry/v2/blobs/sha256/ab/abe5af443833d60cf672e2ac57589410dddec060ed725d3e676f1865af63d2e2/data: EOF" err.message="unknown error" http.request.method=PUT
+...
+time="2016-04-02T07:01:46.056520049-04:00" level=error msg="error putting into main store: s3: The request signature we calculated does not match the signature you provided. Check your key and signing method." http.request.method=PUT
+atest
+----
+
+If you see such errors, please contact your Amazon S3 support. There may be a
+problem in your region or with your particular bucket. Let us know in case
```

Comment:
I would consider removing "Let us know in case the problem persists" because they are already directed to contact support.